.github/workflows/tests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r test-requirements.txt
+        pip install -r requirements.txt -r test-requirements.txt
```

Comment:
this is not correct -- this bypasses your library and does not test that it is packaged properly (the root cause of why this broke in the first place!)

the editable install which was here before is the right way to do this